src/routes/common/logging/logging.service.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { ClsService } from 'nestjs-cls';
```

Comment:
> Is there a reason this test file is not inside `__tests__` but the module test file is? Is it just our convention?

Our convention in the CGW was to have `__tests__` as more like test-scoped utilities. I see that this can be confusing. But the goal was to not share the same path between what should only be used in a testing scenario vs production.

Open to suggestions regarding this one!

Reply:
Ah that makes sense! Thanks